src/main/kotlin/calculator/Token.kt

```diff
@@ -0,0 +1,20 @@
+package calculator
+
+sealed interface Token
+
+class NumberToken(token: String) : Token {
+    val value = token.toInt()
+}
+
+class OperatorToken(token: String) : Token {
+    private val operatorType: OperatorType = OperatorType.of(token)
+
+    fun compute(a: Int, b: Int): Int {
+        return when (operatorType) {
+            OperatorType.PLUS -> a + b
+            OperatorType.SUBTRACT -> a - b
+            OperatorType.MULTIPLY -> a * b
+            OperatorType.DIVIDE -> a / b
+        }
```

Comment:
사칙연산이 아닌 다른 operatorType 이 들어온 경우도 명시적인 처리를 해주면 좋을 것 같습니다. 😀
```suggestion
    fun compute(a: Int, b: Int): Int {
        return when (operatorType) {
            OperatorType.PLUS -> a + b
            OperatorType.SUBTRACT -> a - b
            OperatorType.MULTIPLY -> a * b
            OperatorType.DIVIDE -> a / b
            else -> { throw IllegalArgumentException("사칙연산이 아닙니다.") }
        }
```

Reply:
else 예약어를 안쓰려고 노력하다보니... 에러 처리는 OperatorType의 of 메서드에서 throw하도록 처리해보았습니다!

Reply:
앗 그렇네요 ㅎㅎ 잘못된 리뷰를 드렸는데 정정해주셔서 감사합니다.